docs/juttle_ecosystem.md

```diff
@@ -0,0 +1,19 @@
+# The Juttle Ecosystem
+
+![Juttle Ecosystem](../docs/images/JuttleEcosystemDiagram.png)
+
+The Juttle Ecosystem consists of the following projects:
+
+* [juttle](https://github.com/juttle/juttle) contains the core bits of the language, including the compiler and Javascript runtime for executing Juttle programs, as well as a simple CLI that enables interactive executions using console-based visualizations.
```

Comment:
The description at the top of the README now says:

> This repository contains the core Juttle compiler, the Javascript runtime, a set of basic adapters to connect to files or http sources, and a command line interface with text-based and tabular views.

I think we should clone that text here to be consistent.